src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$day-gap};
+  width: #{$calendar-width};
+  padding: 70px 10px;
+
+  &__day {
+    box-sizing: border-box;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: #{$box-day-height};
+    width: #{$box-day-width};
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+
+    border: 1px solid black;
+    background-color: #{$main-color};
+
+    transition: transform 0.5s ease-in-out;
```

Comment:
CL 8 - To make the background-color also change smoothly, you need to add it to the `transition`

Reply:
ok,
I will change.